packages/@lwc/engine/src/framework/main.ts

```diff
@@ -8,11 +8,12 @@
 // TODO [#1296]: Revisit these exports and figure out a better separation
 export { createContextProvider } from './context-provider';
 export { getComponentDef, isComponentConstructor } from './def';
-export { BaseLightningElement as LightningElement } from './base-lightning-element';
+export { BaseLightningElement } from './base-lightning-element';
```

Comment:
Let's expose directly `BaseLigthningElement` as `LightningElement` out of `engine-core`.